src/03-control-flow.js

```diff
@@ -86,7 +114,15 @@ function positiveNegativeOrZero(a) {}
  *  aroundTheWorldGreeting();
  *  //> "Hello World"
  */
-function aroundTheWorldGreeting(language) {}
+function aroundTheWorldGreeting(language) {
+  if(language === "german") {
+    return `Hallo Welt`
+  } else if ( language === "spanish") {
+    return 'Hola Mundo'
```

Comment:
I actually really like how we do not have a ` case === 'english ` clause.  However technically someone may make the argument that we need that.  Just make sure we know why we do not need to check for both cases in this case.